### DISCUSSION TITLE
PSADT and Intune

### DISCUSSION THREAD

**Discussion by Pollewops** [PSAppDeployToolkit/PSAppDeployToolkit#910] [Q&A]
Convert my PSADT application to .intunewin Win32App format.
Added to Intune and tried to make it installed/uninstalled using Company Portal.

When using command: deploy-application.exe it works, but when using default powershell ps1 command it does not work.

Is there an explanation for this?

**Comment by Pollewops** [comment]
I found the issue. With the exe the program starts in 64bit.
With ps1 it starts in 32bit and failed.

**Reply by mjr4077au** [reply]
@Pollewops I'm sorry but I missed this. To ensure PowerShell starts using the 64-bit interpretor using Intune, always call PowerShell by using `%windir%\sysnative\WindowsPowerShell\v1.0\powershell.exe`. The Intune Management Extension (IME/sidecar) is 32-bit and will always invoke a 32-bit process. The `sysnative` is a bit of magic that 32-bit processes can use to access the 64-bit `system32` directory.
